### ISSUE TITLE
[BUG] [webkit] GestureEvent is undefined in webkit browsers (Mobile Safari, iOS)

### ISSUE THREAD

**Issue by ZitRos** [microsoft/playwright#22735]
TL;DR There's [GestureEvent](https://developer.mozilla.org/en-US/docs/Web/API/GestureEvent#browser_compatibility) which lives in all Apple (webkit) browsers, but it is missing in Playwright.

<img width="960" alt="image" src="https://user-images.githubusercontent.com/4989256/235442616-9ccbf82d-0745-416f-a353-c036fedb428a.png">

### System info

Playwright Version: 1.32.0
Operating System: macOS Ventura 13.2.1
Browser: latest auto-downloaded WebKit
Other info: node v19.6.0

### Source code

For demonstration, `test/dummy.test.ts`:

```ts
import { expect, test } from '@playwright/test';

test('dummy', async ({ page }) => {
  await page.goto('.');

  const isGestureEventDefined = await page.evaluate(async () => {
    return !!window.GestureEvent;
  });

  expect(isGestureEventDefined).toBe(true);
});
```

<details>
  <summary>playwright.config.ts</summary>
  
  ```typescript
  import { defineConfig, devices } from '@playwright/test';

  export default defineConfig({
    testDir: 'test',
    fullyParallel: false,
    retries: 0,
    workers: 1,
    use: {
      baseURL: 'http://x.com',
      trace: 'on-first-retry',
    },
    projects: [
      {
        name: 'webkit',
        use: { ...devices['Desktop Safari'] },
      },
    ],
  });

  ```
  
</details>

**Steps**

Run the test and see that it fails.

**Expected**

Test is passing.

**Actual**

Test is failing.

# THE GOAL

Some JavaScript library code may test the presence of `GestureEvent` or `safari` on `window` to detect Safari/Mobile Safari browser. Playwright doesn't expose `GestureEvent` and hence the detection fails.

P.S. User-Agent detection doesn't work as reliably as `GestureEvent`, hence `webkit` UA is present in some other browsers, UA string presence may change over time or it can be patched relatively simply.

**Comment by ZitRos** [comment]
Thank you a ton @yury-s! 🚀
